src/app/(marketing)/profile/[username]/page.tsx

```diff
@@ -0,0 +1,195 @@
+import { CheckMark } from "@/components/CheckMark"
+import {
+  Layout,
+  LayoutBanner,
+  LayoutFooter,
+  LayoutHeader,
+  LayoutHero,
+  LayoutMain,
+} from "@/components/Layout"
+import { Anchor } from "@/components/ui/Anchor"
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/Avatar"
+import { AvatarGroup } from "@/components/ui/AvatarGroup"
+import { Badge } from "@/components/ui/Badge"
+import { Button } from "@/components/ui/Button"
+import { Card } from "@/components/ui/Card"
+import { Progress } from "@/components/ui/Progress"
+import { Separator } from "@/components/ui/Separator"
+import { Skeleton } from "@/components/ui/Skeleton"
+import { Schemas } from "@guildxyz/types"
+import { ArrowRight } from "@phosphor-icons/react/dist/ssr"
+import { ActivityChart } from "../_components/ActivityChart"
+import { CircularProgressBar } from "../_components/CircularProgressBar"
+import { ContributionCard } from "../_components/ContributionCard"
+import { EditContributions } from "../_components/EditContributions"
+import { EditProfile } from "../_components/EditProfile"
+import { LevelBadge } from "../_components/LevelBadge"
+import { OperatedGuildCard } from "../_components/OperatedGuildCard"
+import { RecentActivity } from "../_components/RecentActivity"
+
+async function getProfileData(username: string) {
+  const req = `https://api.guild.xyz/v2/profiles/${username}`
+  try {
+    const res = await fetch(req)
+    if (!res.ok) {
+      throw new Error("Failed to fetch profile data")
+    }
+    return res.json() as Promise<Schemas["Profile"]>
+  } catch (e) {
+    // mocking for the time being if fetch fails
+    console.error(e)
+    const res = {
+      id: 4,
+      userId: 6027190,
+      username: "durumm",
+      name: "durum",
+      bio: null,
+      profileImageUrl:
+        "https://guild-xyz.mypinata.cloud/ipfs/QmWGdo6FkjSz22oDZFMJysx3hGKoVqtzTWVMx9tTSP7jvi",
+      backgroundImageUrl: null,
+      createdAt: "2024-07-25T10:04:20.781Z",
+      updatedAt: "2024-07-25T10:04:20.781Z",
+    }
+    return res
+  }
+}
+
+const Page = async ({
+  params: { username },
+}: {
+  params: { username: string }
+}) => {
+  const profile = await getProfileData(username)
+  const level = 50
+
+  return (
+    <Layout>
+      <LayoutHero>
+        <LayoutHeader />
+        <LayoutBanner className="-bottom-[500px]">
```

Comment:
Let's use the `Layout` component's capabilities here! The background image should be dynamic (its height basically depends on the profile description), so in my opinion we should remove this arbitrary value here and just put the image, name, handle, description, followers section inside `LayoutHero`, that way the CSS will handle the proper height. ;)